libyang/schema.py

```diff
@@ -1899,6 +1984,57 @@ def extensions(self) -> Iterator["ExtensionParsed"]:
             yield ExtensionParsed(self.context, ext, self.module)
 
 
+# -------------------------------------------------------------------------------------
+class PIdentity:
```

Comment:
It would be nicer if we could use the register mechanism like for the other node types. Is there a problem with that?

Reply:
PIdentity is not PNode based. Even the underlying C structure those are separated.